Sources/CloudFoundryEnv/JSONUtils.swift

```diff
@@ -28,7 +28,7 @@ public struct JSONUtils {
   public static func convertStringToJSON(text: String?) -> [String:Any]? {
     let data = text?.data(using: String.Encoding.utf8)
     guard let nsData = data else {
-      Log.error("Could not generate JSON object from string: \(text)")
+      Log.error("Could not generate JSON object from string: \(String(describing: text))")
```

Comment:
Because this line is in the `else` block of the guard statement, this means that `nsData` is nil at this point of the code.